tests/rptest/tests/redpanda_startup_test.py

```diff
@@ -23,3 +26,48 @@ def __init__(self, test_context):
     @cluster(num_nodes=3)
     def test_startup(self):
         pass
+
+
+class RedpandaFIPSStartupTest(RedpandaTest):
+    """
+    Tests that Redpanda can start up in FIPS permissive mode
+    """
+    def __init__(self, test_context):
+        super(RedpandaFIPSStartupTest,
+              self).__init__(test_context=test_context)
+
+        for node in self.redpanda.nodes:
+            self.redpanda.set_extra_node_conf(
+                node, {
+                    "fips_mode":
+                    "permissive",
+                    "openssl_config_file":
+                    self.redpanda.get_openssl_config_file_path(),
+                    "openssl_module_directory":
+                    self.redpanda.get_openssl_modules_directory()
+                })
+
+    @cluster(num_nodes=3)
+    def test_startup(self):
```

Comment:
Maybe parameterize by mode, adding testing capability to ensure it crashes when expected in `enabled` mode as well

Reply:
Filed as a todo: https://redpandadata.atlassian.net/browse/CORE-3194